frontPage.html

```diff
@@ -0,0 +1,310 @@
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="styles.css" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <title>UNiQUe UI</title>
+</head>
+
+<body>
+
+  <nav class="navigation container ">
+    <br />
+    <div class="nav-developer">UNiQUe UI</div>
+    <ul class="list-non-bullet nav-pills">
+      <li class="list-item-inline"><a class="link link-active" href="/index.html">home</a></li>
+      <li class="list-item-inline"><a class="link " href="/frontPage.html">docs</a></li>
+      <li class="list-item-inline"><a class="link " href="https://github.com/ruchikasrivastava123/Component_Library">github</a></li>
+    </ul>
+  </nav>
+  <div class="main-div">
+    <nav class="nav2">
+      <ul class="list-non-bullet-nav2">
+        <li class="nav2-li"><a href="#Avatar" class="a-nav2" >Avatar</a></li>
+        <li class="nav2-li"><a href="#Alert" class="a-nav2" >Alert</a></li>
+        <li class="nav2-li"><a href="#Badge" class="a-nav2" >Badge</a></li>
+        <li class="nav2-li"><a href="#Button" class="a-nav2">Button</a></li>
+        <li class="nav2-li"><a href="#Card" class="a-nav2">Card</a></li>
+        <li class="nav2-li"><a href="#Image" class="a-nav2">Image</a></li>
+        <li class="nav2-li"><a href="#Input" class="a-nav2">Input</a></li>
+        <li class="nav2-li"><a href="#Lists" class="a-nav2">Lists</a></li>
+        <li class="nav2-li"><a href="#Navigation" class="a-nav2">Navigation</a></li>
+        <li class="nav2-li"><a href="#Modal" class="a-nav2">Modal</a></li>
+        <li class="nav2-li"><a href="#Rating" class="a-nav2">Rating</a></li>
+        <li class="nav2-li"><a href="#SimplifiedGrid" class="a-nav2">Simplified Grid</a></li>
+        <li class="nav2-li"><a href="#Typography" class="a-nav2">Typography</a></li>
+        <li class="nav2-li"><a href="#Toast" class="a-nav2" >Toast</a></li>
+        <li></li>
+        <li></li>
+      </ul>
+    </nav>
+    <div class="sec-div">
+      
+        <div class="sub-div" id="Avatar">
+          <h2 class="div-h2" >Avatar</h2>
+          <img src="images/avatar.svg" alt="image is not supportable" class="avatar-large" />
+          <img src="images/avatar.svg" alt="image is not supportable" class="avatar-medium" />
+          <img src="images/avatar.svg" alt="image is not supportable" class="avatar-small" />
+          <img src="images/avatar.svg" alt="image is not supportable" class="avatar-extra-small" />
+          <br />
+          <img src="images/code-avatar.png" alt="image is not supportable" class="img-div" />
+        </div>
+      <br />
+      <div class="sub-div" id="Alert">
+        <h2 class="div-h2" >Alert</h2>
+        <div class="alert-div">
+          <div class="alert1">&nbsp;&nbsp;&nbsp;&nbsp;Yeah! Successfully submitted
```

Comment:
It's not a good practice to use `&nbsp` more. It makes the code unreadable though it looks cool on view. A better approach will be using padding or margin.